sphinxcontrib/images.py

```diff
@@ -265,13 +265,15 @@ def download_images(app, env):
                       .format(src, dst))
 
 
-def configure_backend(app):
-    global DEFAULT_CONFIG
+def check_config(app, config):
```

Comment:
I'm not sure `check_config()` is the best name for this function since it does not check anything, it updates a dictionary afaik. `configure_backend()` is not a good name either, though, you are right. Maybe `update_config()`?

Reply:
I'm fine with `update_config` since it validates it and updates it as needed.